R/page_size.R

```diff
@@ -87,16 +87,14 @@ page_dim <- function(page_type) {
 #' @return named list with `cpi` and `lpi`, the characters and lines per
 #' inch, respectively.
 #'
+#' @keywords internal
```

Comment:
also this I think it is better to have close to the function declaration

Reply:
ok moved